packages/base-driver/lib/jsonwp-proxy/proxy.js

```diff
@@ -249,7 +249,7 @@ class JWProxy {
         this.downstreamProtocol = this.getProtocolFromResBody(data);
         this.log.info(`Determined the downstream protocol as '${this.downstreamProtocol}'`);
       }
-      if (_.has(data, 'status') && parseInt(data.status, 10) !== 0) {
+      if (this.downstreamProtocol == W3C && _.has(data, 'status') && parseInt(data.status, 10) !== 0) {
```

Comment:
not sure if this is going to work. We don't expect to have the 'status' property on downstream responses if the protocol is W3C